choose-index.md

```diff
@@ -251,7 +253,71 @@ mysql> EXPLAIN SELECT /*+ use_index_merge(t3, idx) */ * FROM t3 WHERE ((a=1 AND
+| id                              | estRows | task      | access object                                                              | operator info                                                                                                                                  |
++---------------------------------+---------+-----------+----------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------+
+| Selection_5                     | 8.00    | root      |                                                                            | json_memberof(cast(1, json BINARY), json_extract(test.t.j1, "$.path")), json_memberof(cast(2, json BINARY), json_extract(test.t.j2, "$.path")) |
+| └─IndexMerge_9                  | 0.01    | root      |                                                                            | type: union                                                                                                                                    |
+|   ├─IndexRangeScan_6(Build)     | 10.00   | cop[tikv] | table:t, index:k1(cast(json_extract(`j1`, _utf8'$.path') as signed array)) | range:[1,1], keep order:false, stats:pseudo                                                                                                    |
+|   └─Selection_8(Probe)          | 0.01    | cop[tikv] |                                                                            | eq(test.t.a, 3)                                                                                                                                |
+|     └─TableRowIDScan_7          | 10.00   | cop[tikv] | table:t                                                                    | keep order:false, stats:pseudo                                                                                                                 |
++---------------------------------+---------+-----------+----------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------+
+5 rows in set, 6 warnings (0.01 sec)
+```
+
+Currently only one index can be used, instead of generating a plan like below that can use multiple indexes to access at the same time:
+
+```
+  Selection
+  └─IndexMerge
+    ├─IndexRangeScan(k1)
+    ├─IndexRangeScan(k2)
+    ├─IndexRangeScan(ka)
+    └─Selection
+      └─TableRowIDScan
+```
+
+### Scenarios that multi-valued indexes cannot support
+
+For `OR` conditions composed of multiple expressions, and these expressions are corresponding to multiple different indexes, then multi-valued indexes cannot be used:
```

Comment:
````suggestion
Currently, TiDB only supports using one index to access instead of generating the following plan that uses multiple indexes to access at the same time:

```
Selection
└─IndexMerge
  ├─IndexRangeScan(k1)
  ├─IndexRangeScan(k2)
  ├─IndexRangeScan(ka)
  └─Selection
    └─TableRowIDScan
```

### Unsupported scenarios

For `OR` conditions composed of multiple expressions that correspond to multiple different indexes, multi-valued indexes cannot be used:
````